doc/nrf/working_with_nrf/nrf52/gs.rst

```diff
@@ -42,6 +42,8 @@ On your computer, one of the following operating systems:
 * macOS X, latest version
 * Ubuntu Linux, latest Long Term Support (LTS) version
```

Comment:
Looking at the support table, are these entries now wrong? Tier 1 support is not available for the latest version or LTS version of either, only Tier 2.

Would it actually be better to replace all the bullets with the link to the Supported operating systems section?

Reply:
Yeah, You are right. May be just list the OS name and for versions we could like to the Supported operating systems section like we have in infocenter docs? 
https://infocenter.nordicsemi.com/topic/struct_nrftools/struct/nrftools_nrfconnect.html